app/javascript/elm/tests/MainTests.elm

```diff
@@ -22,6 +23,26 @@ import Test.Html.Selector as Slc
 import TimeRange
 
 
+heatMapThresholdsWithMinimum : Int -> HeatMapThresholds
+heatMapThresholdsWithMinimum value =
+    { threshold1 = { value = value, default = 1 }
+    , threshold2 = { value = 2, default = 2 }
+    , threshold3 = { value = 3, default = 3 }
+    , threshold4 = { value = 4, default = 4 }
+    , threshold5 = { value = 5, default = 5 }
+    }
+
+
+heatMapThresholdsWithMaximum : Int -> HeatMapThresholds
+heatMapThresholdsWithMaximum value =
+    { threshold1 = { value = 1, default = 1 }
+    , threshold2 = { value = 2, default = 2 }
+    , threshold3 = { value = 3, default = 3 }
+    , threshold4 = { value = 4, default = 4 }
+    , threshold5 = { value = value, default = 5 }
```

Comment:
what if value is lower than one or all of the other thresholds?

Reply:
as above